spec/support/time_helpers.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
```

Comment:
I guess I have to look at the directory to know this exists :thinking: rather than cursory look the spec helper

Reply:
I guess its pretty idiomatic